### PR TITLE
best-practices.md: fix typo, prefer building from source

### PR DESCRIPTION
**Pull request by erenon**

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by erenon** [comment]
I signed it!

**Comment by mhlopko** [comment]
Can you rephrase the sentence? We just spent 5 minutes fighting which version is correct (and didn't reach the agreement :) So it's clearly too complicated. Something like If you build from source, you will not have problems such as ...

**Comment by mhlopko** [comment]
Friendly ping...

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?
